build-system/tasks/make-extension/template/bento/extensions/amp-__component_name_hyphenated__/__component_version__/base-element.js

```diff
@@ -36,5 +36,9 @@ BaseElement['layoutSizeDefined'] = true;
 /** @override */
 BaseElement['usesShadowDom'] = true;
 
+// DO NOT SUBMIT: If BaseElement['shadowCss']  is set to `null`, remove the
```

Comment:
```suggestion
// __do_not_submit__: If BaseElement['shadowCss']  is set to `null`, remove the
```
This as-written will probably fail CI